woocommerce-gateway-ebanx/assets/js/advanced-options.js

```diff
@@ -38,15 +44,27 @@
 
   var updateFields = function(){
     var modes = modesField.val();
+    var brazil_val = countryPayments.brazil.val();
+    var chile_val = countryPayments.chile.val();
     disableFields(fields);
+    disableFields(fieldBrazilTaxes);
+
+    if (brazil_val != null && brazil_val.length > 0) {
+      enableFields(fieldBrazilTaxes);
+    }
 
     if (fieldsToggler[0].checked) {
       enableFields(fields.filter('.always-visible'));
-      for (var i in modes) {
-        enableFields(fields.filter('.' + modes[i]));
+      if (brazil_val != null && brazil_val.length > 0) {
+        for (var i in modes) {
+          enableFields(fields.filter('.' + modes[i]));
+        }
+        if (modes.length == 2) {
```

Comment:
Why 2? Is there a automatically way to do this? It's dangerous put this in hard code haha